implementations/java-implementations-pmd-ruleset.xml

```diff
@@ -0,0 +1,315 @@
+<?xml version="1.0"?>
```

Comment:
Sorry, one last nitpick -- I sometimes use `ls implementations/ | wc -l` as a quick way to count how many we support.

Can you hide this file (i.e. rename to `.java-implementations-pmd-ruleset.xml`) so it doesn't cause minor inconvenience (of needing to change to filter directories) there?